2009-10-29  Ulrich Drepper  <drepper@redhat.com>
 
	[BZ #10553]
	* malloc/malloc.c (malloc_info): Fix typos in format strings.
	Patch by John Sullivan <jsrhbz@kanargh.force9.co.uk>.

--- a/malloc/malloc.c
+++ b/malloc/malloc.c
@@ -6474,8 +6474,8 @@ malloc_info (int options, FILE *fp)
   fprintf (fp,
 	   "<total type=\"fast\" count=\"%zu\" size=\"%zu\"/>\n"
 	   "<total type=\"rest\" count=\"%zu\" size=\"%zu\"/>\n"
-	   "<system type=\"current\" size=\"%zu\n/>\n"
-	   "<system type=\"max\" size=\"%zu\n/>\n"
+	   "<system type=\"current\" size=\"%zu\"/>\n"
+	   "<system type=\"max\" size=\"%zu\"/>\n"
 	   "<aspace type=\"total\" size=\"%zu\"/>\n"
 	   "<aspace type=\"mprotect\" size=\"%zu\"/>\n"
 	   "</malloc>\n",
